src/ImmunizationHistory.js

```diff
@@ -75,7 +60,7 @@ function renderImmunizationGroup(
     return (
       <tr key={key}>
         <td>{immunization.occurrenceDateTime}</td>
-        <td>{renderName(immunization.vaccineCode.coding)}</td>
+        <td>{futil.renderCodeableJSX(immunization.vaccineCode, dcr)}</td>
```

Comment:
Awesome work, Sean! I have but one nitpick. It might be nice to have more control over what the safe display text is. For example, in the immunization case here, I think I'd prefer showing the user something like "loading..." rather than the code value itself. I don't think this is super urgent, since the JSON payload is small enough that the flicker when fetching the vaccine names is very quick. I'm just imagining someone loading a SHLink on their phone with poor service and being confused by the code numbers in the vaccine name column for ~3 seconds.